symbol sets/src/ISMTBOM_15000.xmap

```diff
@@ -6728,7 +6728,7 @@ Note: if the stream is wider than 0.25mm, adjust this symbol so it extends 0.5mm
             </symbol>
             <symbol type="4" id="166" code="601.1" name="North lines pattern" is_hidden="true">
                 <area_symbol inner_color="-1" min_area="0" patterns="1">
-                    <pattern type="1" angle="1.5708" rotatable="true" line_spacing="33330" line_offset="0" offset_along_line="0" color="1" line_width="140"/>
+                    <pattern type="1" angle="1.5708" line_spacing="33330" line_offset="0" offset_along_line="0" color="1" line_width="140"/>
```

Comment:
The ISMTBOM north lines pattern was rotatable. But they shall be parallel to "the sides of the map".